components/readwise/sources/new-documents/new-documents.mjs

```diff
@@ -0,0 +1,64 @@
+import readwise from "../../readwise.app.mjs";
+import { DEFAULT_POLLING_SOURCE_TIMER_INTERVAL } from "@pipedream/platform";
+
+export default {
+  key: "readwise-new-documents",
+  name: "New Documents",
+  description: "Emit new document",
+  version: "0.0.1",
+  type: "source",
+  dedupe: "unique",
+  props: {
+    readwise,
+    timer: {
+      label: "Polling interval",
+      description: "Pipedream polls the Readwise API on this schedule",
+      type: "$.interface.timer",
+      default: {
+        intervalSeconds: DEFAULT_POLLING_SOURCE_TIMER_INTERVAL,
+      },
+    },
+    location: {
+      type: "string",
+      label: "Location",
+      description: "(Optional) The document's location, can be one of: new, later, shortlist, archive, feed",
```

Comment:
Oh, do `options` limit the user only to select these options? I think Readwise may add more options in the future, and I do not think we should follow up on their updates and update the component.

Reply:
Not necessarily, users can pass a custom value instead of selecting one of the provided options. It's just meant as a more intuitive option when you have a set of possible values that can be used.

Since the options can change in the future, it'd be great to include a link to the appropriate API documentation in this description.